crossing-repair.lic

```diff
@@ -2,37 +2,184 @@
   Documentation: https://elanthipedia.play.net/Lich_script_repository#crossing-repair
 =end
 
-custom_require.call(%w[common common-money common-travel drinfomon equipmanager])
+custom_require.call(%w[common common-money common-travel equipmanager])
 
 class CrossingRepair
-  include DRC
```

Comment:
Lines 189 and 222 need to become `DRC.right_hand`

Reply:
Good catch. Fixing.